packages/form-js-viewer/src/render/components/Util.js

```diff
@@ -1,25 +1,21 @@
 import snarkdown from '@bpmn-io/snarkdown';
 import { get } from 'min-dash';
+import classNames from 'classnames';
 
 import { sanitizeHTML } from './Sanitizer';
 
-export function formFieldClasses(type, errors = []) {
+export function formFieldClasses(type, { errors = [], disabled = false } = {}) {
   if (!type) {
     throw new Error('type required');
   }
 
-  const classes = [
-    'fjs-form-field',
-    `fjs-form-field-${ type }`
-  ];
-
-  if (errors.length) {
-    classes.push('fjs-has-errors');
-  }
-
-  return classes.join(' ');
+  return classNames('fjs-form-field', `fjs-form-field-${type}`, {
+    'fjs-has-errors': errors.length > 0,
+    'fjs-disabled': disabled,
```

Comment:
```suggestion
    'fjs-disabled': disabled
```

Reply:
Just to follow current code styles, I'm happy to discuss this because of the rationale you gave in another PR: https://github.com/camunda/team-hto/pull/48#discussion_r1028018641

For now, I'd try to be consistent.